integration/apocalypse/index.js

```diff
@@ -0,0 +1,346 @@
+
+// Contracts
+const oracleJson = require('../../evm-contracts/abi/v0.6/Oracle.json')
+const fluxAggregatorJson = require('../../evm-contracts/abi/v0.6/FluxAggregator.json')
+
+// RPC
+const RPC_GETH1  = 'http://localhost:8545'
+const RPC_GETH2  = 'http://localhost:18545'
+const RPC_PARITY = 'http://localhost:28545'
+const RPC_PROVIDERS = [RPC_GETH1, RPC_GETH2]
+
+// Personas
+const accounts = require('./config/accounts.json')
+const carol = new ethers.Wallet(accounts.carol.privkey.substring(2), new ethers.providers.JsonRpcProvider(RPC_GETH1))
+const loki = new ethers.Wallet(accounts.loki.privkey.substring(2), new ethers.providers.JsonRpcProvider(RPC_GETH1))
+const lokiJr = new ethers.Wallet(accounts['loki-jr'].privkey.substring(2), new ethers.providers.JsonRpcProvider(RPC_GETH2))
+const geth1 = new ethers.Wallet(accounts.geth1.privkey.substring(2), new ethers.providers.JsonRpcProvider(RPC_GETH1))
+const geth2 = new ethers.Wallet(accounts.geth2.privkey.substring(2), new ethers.providers.JsonRpcProvider(RPC_GETH2))
+const parity = new ethers.Wallet(accounts.parity.privkey.substring(2), new ethers.providers.JsonRpcProvider(RPC_GETH2))
+const neil = new ethers.Wallet(accounts.neil.privkey.substring(2), new ethers.providers.JsonRpcProvider(RPC_GETH2))
+const nelly = new ethers.Wallet(accounts.nelly.privkey.substring(2), new ethers.providers.JsonRpcProvider(RPC_GETH2))
+
+const EXTERNAL_ADAPTER_URL = 'http://external_adapter:6644'
+const CHAINLINK_URL_NEIL  = 'http://localhost:6688'
+const CHAINLINK_URL_NELLY = 'http://localhost:6689'
+
+async function main() {
+    console.log('Awaiting Geth DAG generation...')
+    await gethDAGGenerationFinished()
+
+    console.log('Deploying Flux Monitor contracts...')
+    let { linkToken, fluxAggregator } = await deployFluxMonitorContracts(carol, [ neil.address, nelly.address ])
+
+    console.log('Initializing price feed...')
+    await setPriceFeedValue(EXTERNAL_ADAPTER_URL, 100)
+
+    console.log('Initiating transaction tornado...')
+    await mimicRegularTraffic(200, RPC_PROVIDERS)
+
+    console.log('Waiting for things to get really bad...')
+    await sleep(20000)
+
+    console.log('Adding job specs to Chainlink nodes...')
+    let envNeil = { CLIENT_NODE_URL: CHAINLINK_URL_NEIL, ROOT: '/clroot' }
+    await run(`chainlink -j admin login -f /apicredentials`, envNeil)
+    await run(`chainlink -j jobs create '${JSON.stringify(makeJobSpecFluxMonitor(fluxAggregator.address, EXTERNAL_ADAPTER_URL))}'`, envNeil)
+    let envNelly = { CLIENT_NODE_URL: CHAINLINK_URL_NELLY, ROOT: '/clroot' }
+    await run(`chainlink -j admin login -f /apicredentials`, envNelly)
+    let jobSpec = JSON.parse(await run(`chainlink -j jobs create '${JSON.stringify(makeJobSpecFluxMonitor(fluxAggregator.address, EXTERNAL_ADAPTER_URL))}'`, envNelly))
+
+    await successfulJobRuns(jobSpec.id, 1, envNeil)
+    await successfulJobRuns(jobSpec.id, 1, envNelly)
+    fluxAggregator.
```

Comment:
something is missing here!

Reply:
This file has been superseded by `scenarios/flux-monitor.js`.  Oversight on my part.